conf/spark-env.sh.template

```diff
@@ -79,3 +80,6 @@
 # Options for beeline
 # - SPARK_BEELINE_OPTS, to set config properties only for the beeline cli (e.g. "-Dx=y")
 # - SPARK_BEELINE_MEMORY, Memory for beeline (e.g. 1000M, 2G) (Default: 1G)
+
+# Options for spark-submit, spark-shell, pyspark, run-example, sparkR
```

Comment:
Does `pyspark` mean PySpark shell? Let's either explicitly use the script name like `bin/...` or use the full name as PySpark shell.

Reply:
Updated to use `./bin/...`